source/indexes.txt

```diff
@@ -12,6 +12,28 @@ Manage Indexes
    :depth: 1
    :class: singlecol
 
+Indexes are special data structures that support the efficient 
+execution of queries in MongoDB. Indexes store a portion of a 
```

Comment:
Suggest making this last sentence (borrowed from MongoDB manual):

```
Indexes store a portion of a  collection's data in an easy-to-traverse form. The index stores
the value of a specific field or set of fields, ordered by the value of the field.
```

This way, when we do the bullets mentioned in my other comment, we give context for why the sort operations are pre-sorted.